build.sh

```diff
@@ -42,6 +42,8 @@ BUILD_DIRS="${LIBRMM_BUILD_DIR} ${RMM_BUILD_DIR}"
 VERBOSE_FLAG=""
 BUILD_TYPE=Release
 INSTALL_TARGET=install
+BUILD_BENCHMARKS=OFF
```

Comment:
We should add these new arguments to the help documentation

Reply:
Done.